src/software/proto/primitive/primitive_msg_factory.h

```diff
@@ -138,3 +130,12 @@ std::unique_ptr<TbotsProto::Primitive> createAutokickMovePrimitive(
  * @return Pointer to Stop Primitive Message
  */
 std::unique_ptr<TbotsProto::Primitive> createStopPrimitive(bool coast);
+
+/**
+ * Convert dribbler mode to dribbler speed
+ *
+ * @param dribbler_mode The DribblerMode
+ *
+ * @return the dribbler speed
```

Comment:
Think it would be nice if we had units for returning dribbler speed just to keep it consistent.

Reply:
added